app/page.tsx

```diff
@@ -1,5 +1,5 @@
-import WorkInProgress from '@/components/work-in-progress';
+import WorkInProgress from "@/components/work-in-progress";
 
 export default function Home() {
-  return <WorkInProgress />;
-}
+  return <WorkInProgress/>;
```

Comment:
what, stop editing this file

Reply:
this is not resolved, why marked as resolved. if you're confused, ask me, stop fighting with Git.